### DISCUSSION TITLE
Planning with selection of 'fast', 'fast and curvy', 'cury', 'very curvy'

### DISCUSSION THREAD

**Discussion by lucvl5** [devemux86/cruiser#140] [Q&A] [answered]
I'm confused: how can i select the planning mode for parts of a route?
I don't find it
Thanks

**Comment by devemux86** [comment, marked as answer]
Hi @lucvl5,

Kurviger routing is not available in Cruiser.

But you can import _offline_ the kurviger routes and navigate them.
Rerouting only calculates until the next waypoint, so the kurviger routes remain unchanged.

**Comment by devemux86** [comment]
You can also use the BRouter (offline) avoidance options, so offline (re)routing) uses the roads you like:

- https://github.com/devemux86/cruiser/discussions/134#discussioncomment-5715695
